portal-impl/src/portal.properties

```diff
@@ -6546,6 +6546,11 @@
     #
     feature.flag.LPS-177031=false
 
+    #
+    # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD__UPPERCASEL__UPPERCASEP__UPPERCASES__MINUS__NUMBER1__NUMBER7__NUMBER7__NUMBER6__NUMBER6__NUMBER4_
+    #
+    feature.flag.LPS-177664=false
+
```

Comment:
@Italo-gabriel can you remove this? Looks like it's causing a merge conflict.

Reply:
SF brought this as modifications, I will try to remove and execute the SF again and see if this will leave the modifications

Reply:
Done !  resolved